apps/api/src/campaign/campaign.service.ts

```diff
@@ -678,25 +780,108 @@ export class CampaignService {
           },
         })
       }
+
+      // TODO - should be managed from the admin UI
+      const percentRaised = (actualAmount / campaign.targetAmount) * 100
+
+      if (percentRaised >= 50 && percentRaised < 90)
+        // Do not await  -> send to background
```

Comment:
Maybe I'm missing something here. The comment above gives a hint that we are not awaiting the actual sending of email (SendGrid is a bit slow), so that the request executes 1-2 seconds faster.  An maybe the suggested comment from you touches on something else that should also happen here